superset-frontend/src/components/Table/cell-renderers/TimeCell/TimeCell.test.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { TimeFormats } from '@superset-ui/core';
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import TimeCell from '.';
+
+const DATE = Date.parse('2022-01-01');
+
+test('renders with default format', async () => {
+  render(<TimeCell value={DATE} />);
+  expect(screen.getByText('2022-01-01 00:00:00')).toBeInTheDocument();
+});
+
+test('renders with custom format', async () => {
+  render(<TimeCell value={DATE} format={TimeFormats.DATABASE_DATE} />);
+  expect(screen.getByText('2022-01-01')).toBeInTheDocument();
+});
+
+test('renders with number', async () => {
```

Comment:
@michael-s-molina can we add tests that pass a mixture of null | undefined| invalid time formats as we as mix of invalid DATE values to ensure the cell renderer does not throw runtime error and break app in case a single row has bad / missing data?  I think we want to make cell renderers fault tolerant especially when doing error prone translations like dates.  antd does not give us an error protection on cell rendering so we have to make sure our cell renderers do not throw error that will break the render loop for entire table component / app

Reply:
Good idea. Added ✅